core/document/src/document.rs

```diff
@@ -300,6 +300,14 @@ impl Document {
 				let children = self.layer_panel(path.as_slice())?;
 				Some(vec![DocumentResponse::DocumentChanged, DocumentResponse::ExpandFolder { path, children }])
 			}
+			Operation::ToggleVisibility { path } => {
```

Comment:
@TrueDoctor did we decide we generally want to do `set` or `toggle`?

Reply:
For now toggle